### PR TITLE
feat: if module list is not in sync with CSV file an issue is created

### PR DESCRIPTION
**Pull request by rahalan**
This code includes a pipeline, that runs daily and checks, if the module list drop down (for module issues) is up-to-date. If not, an issue is created.

### CODE REVIEW

File: .github/workflows/avm.platform.sync-avm-modules-list.yml [hunk 1]
```diff
@@ -0,0 +1,36 @@
+name: "avm.platform.sync-avm-modules-list"
```
Comment (AlexanderSehr) [inline .github/workflows/avm.platform.sync-avm-modules-list.yml:1]:
```suggestion
name: .Platform - Sync AVM module list
```

in alignment with changes that will be introduced by #1411

File: .github/workflows/avm.platform.sync-avm-modules-list.yml [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+  sync-avm-modules-list:
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+    steps:
+      - name: "Checkout"
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+      - uses: tibdex/github-app-token@v2
```
Comment (eriqua) [inline .github/workflows/avm.platform.sync-avm-modules-list.yml:18]: Are we good with leveraging external actions in this repo?

Comment (rahalan) [reply .github/workflows/avm.platform.sync-avm-modules-list.yml:18]: @eriqua yes. We discussed with Alex already. This action is already used by Jack and Chris.

Comment (ChrisSidebotham) [reply .github/workflows/avm.platform.sync-avm-modules-list.yml:18]: Resolving as already addressed

**Review by ChrisSidebotham — APPROVED** [review]

**Pull request merged**